management/models.py

```diff
@@ -1,14 +1,22 @@
-from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.models import AbstractUser, Group
 from django.db import models
-from guardian.shortcuts import assign_perm
+from guardian.shortcuts import assign_perm, get_anonymous_user
 
 
 class User(AbstractUser):
-    """
-    Implement a custom user model to add flexibility in the future.
+    """Custom user model.
+
+    All users are given staff status and added to the standard group.
+
     """
 
-    pass
+    def save(self, *args, **kwargs):
+        if self.username != "AnonymousUser":
+            self.is_staff = True
+        super().save(*args, **kwargs)
+        if self.username != "AnonymousUser":
```

Comment:
I think that this doesn't work because this function is called when the anonymous user is created, but as it doesn't exist yet the get_anonymous_user function fails

Reply:
I've done it a different way